src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/OverloadResolution.cs

```diff
@@ -1698,6 +1711,85 @@ private int GetTheBestCandidateIndex<TMember>(ArrayBuilder<MemberResolutionResul
             return currentBestIndex;
         }
 
+        private static void RemoveLowerPriorityMembers<TMemberResolution, TMember>(ArrayBuilder<TMemberResolution> results)
+            where TMemberResolution : IMemberResolutionResultWithPriority<TMember>
+            where TMember : Symbol
+        {
+            // - Then, the reduced set of candidate members is grouped by declaring type. Within each group:
+            //     - Candidate function members are ordered by *overload_resolution_priority*.
+            //     - All members that have a lower *overload_resolution_priority* than the highest found within its declaring type group are removed.
+            // - The reduced groups are then recombined into the final set of applicable candidate function members.
+
+            switch (results)
+            {
+                // Can't prune anything unless there's at least 2 candidates
+                case []:
+                case [_]:
+                // We only look at methods and indexers, so if this isn't one of those scenarios, we don't need to do anything.
```

Comment:
>// We only look at methods and indexers, so if this isn't one of those scenarios, we don't need to do anything. [](http://example.com/codeflow?start=16&length=111)

Is this comment still meaningful since we're no longer directly checking for methods and indexers?